frontend/src/components/MyNavbar/MyNavbar.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+import { 
+    Collapse, 
+    Navbar, 
+    NavbarToggler, 
+    Nav, 
+    NavItem, 
+    NavLink, 
+    NavbarBrand 
+} from 'reactstrap';
+import { navlinks } from "../../global";
+
+/**
+ * Makes the navbar for navigating different pages
+ * of the website.
+ * 
+ * @return {JSX.Element} navbar.
+ */
+export function MyNavbar() {
```

Comment:
There is no default export here.

Reply:
Do you want me to make MyNavbar have a default export?

Reply:
Yes that would be a good idea.